src/graphql/types/abstract/transaction.ts

```diff
@@ -25,11 +25,9 @@ export const transactionInterfaceFields = {
   },
   settlementAmount: {
     type: GT.NonNull(SatAmount),
-    //TODO: Does the amount include fees.
```

Comment:
this would still be relevant information to give to the user as part of the documentation